src/setup-dotnet.ts

```diff
@@ -29,7 +29,14 @@ export async function run() {
     }
 
     if (version) {
-      const dotnetInstaller = new installer.DotnetCoreInstaller(version);
+      const includePrerelease: boolean = Boolean(
+        core.getInput('include-prerelease') || false
```

Comment:
Hi @GGG-KILLER, sorry for the delay, could you please improve this expression? It will be `true` with any value of `include-prerelease` option at the moment. For example, it can be done:
```
(core.getInput("include-prerelease") || "false").toLower() == "true"
```